SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -1684,33 +1690,59 @@ USpatialActorChannel* ActorSystem::TryRestoreActorChannelForStablyNamedActor(AAc
 
 void ActorSystem::InvokeRepNotifies()
 {
-	for (FObjectRepNotifies& ObjectRepNotifies : RepNotifiesToSend)
+	// We have two lists of ObjectRepNotifies, one for Actors and one for Subobjects
```

Comment:
I like . , and ; in comments, but not a big deal

Reply:
```suggestion
	// We have two lists of ObjectRepNotifies, one for Actors and one for Subobjects.
```